perun-base/src/main/java/cz/metacentrum/perun/core/api/exceptions/AttributeNotExistsException.java

```diff
@@ -2,21 +2,34 @@
 
 
 /**
- * Attribute not exists in underlaying data source.
+ * This exception is thrown when the attribute does not exist in underlying data source.
```

Comment:
This one is a bit tricky... It means the attribute definition not exist in the database. If the attribute is just not set for given entity (e.g. the exception is thrown from getUserAttribute(user, "attribute name")) it doesn't raise this exception but return the attribute with null value instead. 

Can you please formulate this to the javadoc?